vimeo-networking/src/main/java/com/vimeo/networking/model/VimeoAccount.java

```diff
@@ -95,13 +108,12 @@ public User getUser() {
         return this.user;
     }
 
-    public void setUser(User user) {
+    public void setUser(@Nullable User user) {
         this.user = user;
     }
 
     @Nullable
-    public String getUserJSON() // For AccountManager.userData [AH]
-    {
+    public String getUserJSON() {
         if (this.user == null) {
             return null;
         }
```

Comment:
The logic for this method seems flawed. I realize that it isn't part of this PR, but if the user ever changes on this account object between calls to this method it will be returning an incorrectly cached userJSON string.